lib/ex_sanity/cms.ex

```diff
@@ -0,0 +1,39 @@
+defmodule ExSanity.CMS do
+  @moduledoc """
+  Inspired by Ecto.Repo. Runs queries against the Sanity CMS and returns the
+  result.
+
+  Currently only supports "read" type queries. Create/update queries are on
+  the roadmap.
+  """
+  alias ExSanity.Client
+  alias ExSanity.Query
+
+  def get(query, id) do
+    query_string = Query.build(query, id)
+
+    {:ok, response} = Client.query(query_string)
+
+    [document | _] = response.body["result"]
+
+    document
+  end
+
+  def one(query) do
+    query_string = Query.build(query)
+
+    {:ok, response} = Client.query(query_string)
+
+    [document | _] = response.body["result"]
+
+    document
+  end
```

Comment:
Ecto's `one/1` raises if there's more than one result, whereas this one returns the first result from the set. Would it make more sense to rename the function to `first` for accuracy?